server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -356,6 +356,12 @@ export class TransformHandler {
             }
         }
         this.logging.log('poll : returning response from server : ' + JSON.stringify(response))
+        if (failureStates.includes(status)) {
+            this.logging.log(
+                `Transformation job for job ${request.TransformationJobId} is ${status} due to "${response.transformationJob.reason}". 
+                Please close Visual Studio, delete the bin and obj folders for the projects, and try running the transformation again.`
```

Comment:
Update for next time: 

1. Suggestion:  `Please close Visual Studio, delete the directories where build artifacts are generated (e.g. `bin` and `obj`), and try running the transformation again.`
2. We should only suggest the above when the failure reason is "Build validation failed..."
3. Consider making a helper function called `LogSuggestionForFailureResponse(request, response, status)`. There are probably other places where we log some suggestion after a failure message and this will keep the `if-else` logic contained in another function.